helper_functions.py

```diff
@@ -3,8 +3,8 @@
 
 
 class APINames:
-    API_BASE = "http://localhost:8000"
-    # API_BASE = "https://em4lp8.deta.dev"
+    # API_BASE = "http://localhost:8000"
+    API_BASE = "https://em4lp8.deta.dev"
```

Comment:
I think this should be an environment variable as per link / article in django repo ...